azure/packages/azure-client/src/packageVersion.ts

```diff
@@ -6,4 +6,4 @@
  */
 
 export const pkgName = "@fluidframework/azure-client";
-export const pkgVersion = "0.60.1000";
+export const pkgVersion = "1.1.0";
```

Comment:
@sonalivdeshpande did we accidently bumped  azure client in main?

Reply:
> Why [d7ac155](https://github.com/microsoft/FluidFramework/commit/d7ac155fe261e195ce0c5776cd4b8a4685429d51)? was reverted ?

This has conflicts and build failures. opening this commit as a separate PR against next

Reply:
azure client in main is bumped to 1.1.0: https://github.com/microsoft/FluidFramework/pull/10672. I think @tylerbutler would be bumping the azure client in next to 2.0.0 after the release?